src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -204,7 +204,12 @@ def __init__(self, algorithm_name, optimize_mode='minimize'):
         self.json = None
         self.total_data = {}
         self.rval = None
+        self.CL_rval = None
```

Comment:
what's the "CL" means? What's the "self.CL_rval" represent?

Reply:
Constant Liar